.github/workflows/tests.yaml

```
@@ -50,11 +50,20 @@ jobs:
           filters: |
             react:
               - 'packages/react/**'
+            angular:
+              - 'packages/angular/**'
   
       # Run tests for the React package if it has changed
       - name: Run React Tests
         if: steps.changes.outputs.react == 'true'
-        run: pnpm vitest --dom 'packages/react'
+        run:
+          - cd packages/react
+          - pnpm vitest --dom
+      - name: Run Angular Tests
+        if: steps.changes.outputs.angular == 'true'
+        run:
+          - cd packages/react
```

I think for angular tests to run this path would need change
```suggestion
          - cd packages/angular
```